npc/re/instances/OctopusCave.txt

```diff
@@ -64,33 +64,33 @@ mal_dun01,151,235,5	script	Starfish	4_ASTER,{
 		switch (select("Ask what's going on.", "Ask to open the gate.", "Go to other location.")) {
 		case 1:
 			mes("[Starfish]");
-			mes("Lately, our Starfish lady is\r"
-				"suffering with some issues, ooh ooh.");
-			mes("Something bad happen in this\r"
-				"peaceful place, hehe!");
+			mes("Lately, our Starfish lady is");
```

Comment:
i have removed the \r because they said it's messy and change it to 1 liner instead and i think the pr would be merged into 1 commit.